test/core.jl

```diff
@@ -37,4 +37,19 @@ using LinearAlgebra
     @inferred draw!(img, point)
     expected = zeros(Gray{N0f8})
     @test all(img .== expected) == true
+
+    @testset "Thickness" begin
+        point = Point(2,2)
+        img = zeros(Gray{N0f8}, 3, 3)
+        @inferred draw!(img, point, thickness=2)
```

Comment:
Can you add one more simple test on `CartesianIndex{2}` here, to guarantee that `CartesianIndex` works the same as `Point`.

Thanks.